tests/expr_and_series/shift_test.py

```diff
@@ -13,11 +11,8 @@
 }
 
 
-def test_shift(request: Any, constructor: Any) -> None:
-    if "pyarrow_table" in str(constructor):
-        request.applymarker(pytest.mark.xfail)
-
-    df = nw.from_native(constructor(data))
+def test_shift(constructor_eager: Any) -> None:
+    df = nw.from_native(constructor_eager(data), eager_only=True)
```

Comment:
these don't need to change in this test - only in `test_shift_series` did you need to do that, `constructor` is fine here, because we're testing `Expr.shift`
```suggestion
def test_shift(constructor: Any) -> None:
    df = nw.from_native(constructor(data), eager_only=True)
```

Reply:
could you address this comment please?

Reply:
This still needs addressing, could you read over the suggested change more carefully and address it please?